axelor-base/src/main/resources/demo/import_Products/account_accountManagement.csv

```diff
@@ -0,0 +1,973 @@
+importId;company_id;typeSelect;product.importId;tax.code;productFamily.importId;paymentMode.importId;saleAccount_code;saleTax.code;purchaseAccount_code;purchaseTax.code;cashAccount_code;journal_importId;sequence_importId;bankDetails_importId;purchFixedAssetsAccount.code;fixedAssetCategory.importId;interbankCodeLine.importId;financialDiscountAccount.code
```

Comment:
Shouldn't this file be in the Account module ? Otherwise it might not found the according table if the accountable module is not installed